tendrl/node_agent/node_sync/check_all_managed_nodes_status.py

```diff
@@ -1,20 +1,54 @@
 import etcd
 
+from tendrl.commons.objects.job import Job
+import uuid
+
+
+def update_brick_status(fqdn, integration_id):
+    _job_id = str(uuid.uuid4())
+    _params = {
+        "TendrlContext.integration_id": integration_id,
+        "Node.fqdn": fqdn
+    }
+    _job_payload = {
+        "tags": [
+            "tendrl/integration/{0}".format(
+                integration_id
+            )
+        ],
+        "run": "gluster.flows.UpdateBrickStatus",
+        "status": "new",
+        "parameters": _params,
+        "type": "sds"
+    }
+    Job(
+        job_id=_job_id,
+        status="new",
+        payload=_job_payload
+    ).save()
+
 
 def run():
```

Comment:
copy this run() to https://github.com/Tendrl/node-agent/tree/master/tendrl/integrations/gluster/sds_sync/check_all_managed_brick_status.py